rmd_files/reshaping.Rmd

```diff
@@ -169,24 +171,27 @@ head(long_annual_offences)
 
 ---
 
-Essentially, these data are the same as what we started with, but there are some differences we need to iron out in order to get back where we started.
+Essentially, these data are the same as what we started with, but there are some differences.
+```{r, purl=purl_example_code}
+# checking if the original table and working table are identical
+identical(long_annual_offences, annual_offences)
+```
 
 ```{r, purl=purl_example_code}
 head(annual_offences, 3)
 head(long_annual_offences, 3)
 ```
 
-```{r, purl=purl_example_code}
-identical(long_annual_offences, annual_offences)
-```
+In fact, there are *six* differences between these tables. Have a look yourself, and put suggestions in the chat as to what these might be. Then we'll cover how to correct these differences and make our  working table identical to our working table.
```

Comment:
Is there a mistake where it says "make our working table identical to our working table"?

Reply:
good catch, will fix